### DISCUSSION TITLE
Required command line approach to run dependency track.

### DISCUSSION THREAD

**Discussion by spmishra121** [DependencyTrack/dependency-track#1517] [General]
Hi @stevespringett ,

This is just a thought process that if there is any way to run DT using command line to generate FPF report. Because we have Defect dojo to keep all our report.

I am looking that if there is a way to generate the bom using command cyclonedx-bom -o bom.xml. After that those bom can be used by command line in Dependency track environment to generate FPF report.

Is this possible?

Note: I don't want to integrate DT with defect dojo.

**Comment by stevespringett** [comment]
Its likely that one of the tools listed here may work:
https://docs.dependencytrack.org/integrations/community-integrations/

**Comment by valentijnscholten** [comment]
You can get a FPF report via the API via `/project/{uuid}/export`

**Comment by spmishra121** [comment]
Hi @stevespringett ,

In the reference of issue id: 237, I am also facing "No response from server". I was trying to export FPF report using swagger UI.

I am able to authorize successfully but  when running the request to fetch FPF report or looking for all projects, I am getting error...

If something is missing from my side then please guide me.

![image](https://user-images.githubusercontent.com/6615148/169052047-d6981b95-0450-4590-99f0-73643b862955.png)


I am using DT version 4.4.1

**Reply by valentijnscholten** [reply]
The request URL protocol/scheme looks wrong, should be http/https instead of chromeextension. might need to update the api base url somewhere
